### ISSUE TITLE
Upstream the required changes?

### ISSUE THREAD

**Issue by thomaseizinger** [mdvmeijer/fw-fanctrl-nix#1]
I was wondering if you ever attempted to upstream the required changes? They don't look particularly controversial to me :)

**Comment by thomaseizinger** [comment]
Perhaps even the entire nix module can be upstreamed?

**Comment by mdvmeijer** [comment]
Good point, later this week I expect to have some time to look into this
